shared/utilities/ticket-status/test.js

```diff
@@ -9,13 +16,23 @@ describe('getTicketStatusOptions', () => {
       ticketReleaseDate: '2016-07-24T20:30:00+0000',
     };
     const result = getTicketStatusOptions(event);
-    expect(result).to.deep.equal({
```

Comment:
It works perfectly fine when called once, but if called multiple times it starts accumulating dates in the `title`